doc-content/enterprise-only/vehicle-route-planning/run-locally-interactive-proc.adoc

```diff
@@ -0,0 +1,45 @@
+[id='run-locally-interactive-proc_{context}']
+
+= Run the OptaWeb Vehicle Routing runLocally.sh script in interactive mode
+
+Use interactive mode to see the list of downloaded OSM files and country codes assigned to each region.
+You can use the interactive mode to download additional OSM files from Geofabrik without visiting the website and choosing a destination for the download.
+
+
+.Prerequisites
+* OptaWeb Vehicle Routing has been successfully built with Maven as described in xref:download-ref-imp-proc_{context}[].
+* Internet access is available.
+
+.Procedure
+. Change directory to the `{PRODUCT_INIT}-{PRODUCT_VERSION_LONG}-optaweb-vehicle-routing/sources` directory.
```

Comment:
Just small suggestion - remove one directory word

Reply:
Oh, good idea! Did it in all places.